models/Category.py

```diff
@@ -1,22 +1,37 @@
+from sqlalchemy.orm import relationship
+from models.Language import Language
+
 from models.base import db
+from models.LocaleString import LocaleString
 
 
 class Category(db.Model):
+
     __tablename__ = "category"
     id = db.Column(
         db.Integer,
         primary_key=True,
         name="category_id",
         nullable=False
     )
-    name = db.Column(
-        db.String(30),
-        name="category_name",
+    name_id = db.Column(
+        db.String(36),
+        db.ForeignKey("locale_string.string_id"),
+        name="category_name_id",
         nullable=False
     )
+    name = relationship(
+        LocaleString,
+        foreign_keys=[name_id],
+        uselist=True,
+        single_parent=True,
+        cascade="all, delete-orphan"
+    )
 
-    def to_json(self):
+    def to_json(self, locale):
+        name = next((text for text in self.name if text.locale == Language[locale]), None)
```

Comment:
I think we shold check it in utils.json.to_json once and not to check in each model